influxql/functions.gen.go

```diff
@@ -8,12 +8,31 @@ package influxql
 
 // FloatPointAggregator aggregates points to produce a single point.
 type FloatPointAggregator interface {
-	Aggregate(p *FloatPoint)
+	AggregateFloat(p *FloatPoint)
+}
+
+// FloatBulkPointAggregator aggregates multiple points at a time.
+type FloatBulkPointAggregator interface {
+	AggregateFloatBulk(points []FloatPoint)
+}
+
+// AggregateFloatPoints feeds a slice of FloatPoint into an
+// aggregator. If the aggregator is a FloatBulkPointAggregator, it will
+// use the AggregateBulk method.
+func AggregateFloatPoints(a FloatPointAggregator, points []FloatPoint) {
```

Comment:
This function is unused.... Don't we want something like this;

```
func (r *FloatFuncReducer) AggregateFloat(a []FloatPoint) {
    AggregateFloatPoints(r, a)
}
```

Except remove the type switch and it doesn't need to be exported.